pkgs/games/steam/fhsenv.nix

```diff
@@ -111,6 +112,9 @@ in buildFHSUserEnv rec {
     gdk-pixbuf
     fontconfig
 
+    # Hadean Lands dependecies
+    gsettings-desktop-schemas
```

Comment:
Move this to commontargetpkgs and add(don't move) gtk3 there

Reply:
I still haven't gotten it to work.

steam-run has a very similar environment, so I used it to test a few things;

```
nix run -f . steam-run -- bash -c export
(...)
declare -x XDG_DATA_DIRS="/nix/store/lmpz9axrfpir2ig76vdl0nddjr3w53gv-desktops/share:/home/addy/.local/share/flatpak/exports/share:/var/lib/flatpak/exports/share:/home/addy/.nix-profile/share:/etc/profiles/per-user/addy/share:/nix/var/nix/profiles/default/share:/run/current-system/sw/share:/run/opengl-driver/share:/run/opengl-driver-32/share"
```

I.e. `/usr/share` is missing.

`gsettings list-schemas` also doesn't show any schemas

```


The profiles are installed though, so that worked:

```
nix run -f . steam-run -- ls -lah /usr/share/gsettings-schemas
(...)
dr-xr-xr-x 1 nobody nogroup  16 Jan  1  1970 gsettings-desktop-schemas-42.0
dr-xr-xr-x 1 nobody nogroup  16 Jan  1  1970 gtk+3-3.24.33
```
When I add `/usr/share` to `XDG_DATA_DIRS` everything works. Shouldn't it be there by default though?

Reply:
I can confirm @Moredread's findings, I'm currently adding `gsettings-desktop-schemas` and `hicolor-icon-theme` to steam's `extraPkgs`, but file pickers are still not working in my games. I get the same issues as above but using the launch command `XDG_DATA_DIRS="$XDG_DATA_DIRS:/usr/share/" %command%` in addition to adding the two packages fixes it. Weird~